index.bs

```diff
@@ -3253,7 +3253,7 @@ extended attribute’s [=takes a named argument list|named argument list=].
 </div>
 
 Two types are <dfn id="dfn-distinguishable" export>distinguishable</dfn> if
-at most one of the two [=includes a nullable type=]
+at most one of the two [=includes a nullable type=], is a [=dictionary type=], or a [=record type=],
```

Comment:
Agreed. @bzbarsky wanted to avoid leaving a broken spec out for too long, though. Hence the quick fix. Disambiguation can wait for a follow-up PR. Although examples are super useful, I don't think they're appropriate to disambiguate normative statements.